### PR TITLE
109 distance queries

### LINKED ISSUE #109: Add new query classes for distance queries
## Description

The API will now support distance queries, which requires new classes here to interact easily with them. Add the necessary new classes and tests.

### PR DESCRIPTION
**Pull request by coralvanda**
## Description

Adds new query param classes:
  - `APIMilesDistanceQueryParams`
  - `APIDrivingDistanceQueryParams`
  - `APIWalkingDistanceQueryParams`
## What does this address?

Closes #109 

## How has this been tested?

All tests pass.

### CODE REVIEW

**Comment by coralvanda** [comment]
Tests must be rerun after updating API.

**Comment by coralvanda** [comment]
test this please.

File: strato_query/__tests__/test_api_query.py [hunk 1]
```diff
@@ -686,3 +686,49 @@ def test_pure_shape_union_query(self):
...
+        df = self.submit_query(
+            query_params=APIWalkingDistanceQueryParams(
+                start_latitude=41.82937349570897,
+                start_longitude=-71.41338050365448,
+                end_latitude=41.8222540665299,
+                end_longitude=-71.41181409358978))
+        assert 'DISTANCE' in df
+        assert df['DISTANCE'][0] == 0.5761363636, df['DISTANCE'][0]
+        assert 'TIME' in df
+        assert df['TIME'][0] == 17.4333333333, df['TIME'][0]
```
Comment (mjclawar) [inline strato_query/__tests__/test_api_query.py:734]: Need to do almost equals I think for these

Comment (coralvanda) [reply strato_query/__tests__/test_api_query.py]: Darn, you're right. For some reason I thought that wasn't available in this repo. I'll change my rounding to that.

**Pull request merged**